daemon/graphdriver/lcow/lcow.go

```diff
@@ -0,0 +1,278 @@
+		return nil, fmt.Errorf("lcow failed to create '%s': %v", home, err)
+	}
+	return d, nil
+}
+
+// String returns the string representation of a driver. This should match
+// the name the graph driver has been registered with.
+func (d *Driver) String() string {
+	return "lcow"
+}
+
+// Status returns the status of the driver.
+func (d *Driver) Status() [][2]string {
+	return [][2]string{
+		{"LCOW", ""},
+	}
+}
+
+// Exists returns true if the given id is registered with this driver.
+func (d *Driver) Exists(id string) bool {
+	logrus.Debugf("LCOWDriver Exists() id %s", id)
+	_, err := os.Lstat(d.dir(id))
+	return err == nil
+}
+
+// CreateReadWrite creates a layer that is writable for use as a container
+// file system.
+func (d *Driver) CreateReadWrite(id, parent string, opts *graphdriver.CreateOpts) error {
+	logrus.Debugf("LCOWDriver CreateReadWrite() id %s", id)
+	if err := d.Create(id, parent, opts); err != nil {
+		return err
+	}
+	if d.dummyMode {
+		return nil
+	}
+	return createSandbox(d.dir(id))
+}
+
+// Create creates a new read-only layer with the given id.
+func (d *Driver) Create(id, parent string, opts *graphdriver.CreateOpts) error {
+	logrus.Debugf("LCOWDriver Create() id %s", id)
+
+	parentChain, err := d.getLayerChain(parent)
+	if err != nil {
+		return err
+	}
+
+	var layerChain []string
+	if parent != "" {
+		if !d.Exists(parent) {
+			return fmt.Errorf("Cannot create layer with missing parent %s", parent)
+		}
+		layerChain = []string{d.dir(parent)}
+	}
+	layerChain = append(layerChain, parentChain...)
+
+	layerPath := d.dir(id)
+	logrus.Debugf("LCOWDriver Create id %s layerPath %s", id, layerPath)
+	// Make sure the layers are created with the correct ACL so that VMs can access them.
+	if err := system.MkdirAllWithACL(layerPath, 755, system.SddlNtvmAdministratorsLocalSystem); err != nil {
+		return err
+	}
+
+	if err := d.setLayerChain(id, layerChain); err != nil {
+		if err2 := os.RemoveAll(layerPath); err2 != nil {
+			logrus.Warnf("Failed to remove layer %s: %s", layerPath, err2)
+		}
+		return err
+	}
+	return nil
+}
+
+// Remove unmounts and removes the dir information.
+func (d *Driver) Remove(id string) error {
+	logrus.Debugf("LCOWDriver Remove() id %s", id)
+
+	tmpID := fmt.Sprintf("%s-removing", id)
+	tmpLayerPath := d.dir(tmpID)
+	layerPath := d.dir(id)
+
+	logrus.Debugf("LCOWDriver Remove() id %s layerPath %s", id, layerPath)
+	if err := os.Rename(layerPath, tmpLayerPath); err != nil && !os.IsNotExist(err) {
+		return err
+	}
+
+	if err := os.RemoveAll(tmpLayerPath); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Get returns the rootfs path for the id. This will mount the dir at its given path.
+func (d *Driver) Get(id, mountLabel string) (string, error) {
+	logrus.Debugf("LCOWDriver Get() id %s mountLabel %s", id, mountLabel)
+	// TODO @gupta-ak. graphdriver.Get() needs to return an interface
+	// instead of just a string, since the mount point doesn't exist on the host
+	return "", nil
```

Comment:
Can you please explain a little bit better what the dependency is that we are waiting for so that the implementation can be completed?  From the TODO, its not clear that this is still being worked on and has dependencies on continuity or other projects.